en/mkdocs.yml

```diff
@@ -1428,6 +1428,7 @@ nav:
                     - Distributed Deployment Overview: install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md
                     - Configuring a Distributed API-M Deployment: install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md
                     - Configuring a Distributed API-M Deployment with Traffic Manager Separation: install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup-with-tm-separated.md
+                    - Configuring a Distributed API-M Deployment with Key Manager Separation: install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup-with-km-separated.md
```

Comment:
Let's also remove this document from the distributed deployment listing as this is only applicable to migrated environments which will continue to use Opaque tokens.

Reply:
Hiding it from the left navigation completely is not ideal I think. Users might find it and report issues on the missing left navigation as they will see it like a bug. Shall we add it under the Upgrade section?

![image](https://user-images.githubusercontent.com/19791901/142180540-d21e3ab9-69c0-4564-866e-6e520e805842.png)

We can add it above "Upgrading to Log4j2" in the heirarchy. WDYT?

Reply:
@samgnaniah, I had a chat with @bhathiya. Rather than adding it under this location, shall we add both documents, KM separation + Upgrading to Log4j into a separate new menu saying something like Advanced Configurations or similar? Those 2 documents don't seem to fit with the docs at the same level.

Reply:
@tharikaGitHub We should probably make it upgrade specific, so maybe Advanced Upgrade Configurations or something like that.

Reply:
it's not a "configuration" though.